docs/reference/tests/integration-testing.mdx

```diff
@@ -0,0 +1,175 @@
+Integration tests are important for your systems as they catch system-level issues that unit tests miss, whilst being faster than end-to-end tests.
```

Comment:
Suggestion - Change whilst to american english 'while' or just simple 'and are faster...'